README.md

```diff
@@ -5,69 +5,96 @@ Robotic product handling simulation
 
 ## Getting Started
 
-The project is based on Ubuntu 16.04 and ROS Kinect. To install ROS and Gazebo, the following links might be helpful.
-```
-http://wiki.ros.org/kinetic/Installation/Ubuntu
-
-http://gazebosim.org/tutorials?tut=install_ubuntu&cat=install
-```
-Be sure to install gazebo_ros_packages.
-```
-http://gazebosim.org/tutorials?tut=install_ubuntu&cat=install
-```
+The project is based on Ubuntu 16.04, ROS Kinect, gazebo 7.0, wstool and catkin tools. Links to the instructions for installing ROS, Gazebo, etc. are provided below
+- [Install ROS](http://wiki.ros.org/kinetic/Installation/Ubuntu)
+- [Install gazebo 7.0](http://gazebosim.org/tutorials?tut=install_ubuntu&cat=install)
+- [Install wstool](http://wiki.ros.org/wstool#Installation)
+- [Install catkin-tools](http://catkin-tools.readthedocs.io/en/latest/installing.html)
+
 ### Installation
+- Source the ROS environment script
+	
+	```
+	source /opt/ros/kinetic/setup.bash
+	```
+- Create a "gilbreth_ws" catkin workspace
+ 
+	```
+	mkdir -p ~/gilbreth_ws/src
+	cd ~/gilbreth_ws/
+	catkin init
+	```
+- Fetch the gilbreth source code:
+
+	```
+	cd ~/gilbreth_ws/src
+	wstool init .
+	wstool merge https://raw.githubusercontent.com/swri-robotics/gilbreth/kinetic-devel/gilbreth.rosinstall
+	wstool update
+	rosdep install --from-paths . --ignore-src -y
+	```
+- Build the workspace
 
-Create a catkin workspace.
-```
-http://wiki.ros.org/catkin/Tutorials/create_a_workspace
-```
-Second, go to src folder in your workspace:
-```
-cd ~/catkin_ws/src
-```
-Third, download the Gilbreth and dependent packages:
-```
-wstool init .
-wstool merge https://github.com//gilbreth/kinetic-devel/gilbreth.rosinstall
-wstool update
-rosdep install --from-paths . --ignore-src -y
-```
-Make the files
-```
-cd ~/catkin_ws
-catkin_make
-```
+	```
+	cd ~/gilbreth_ws
+	catkin build
+	```
 
 ## Running Gilbreth
 
 ### Setup
-Source the workspace and add the Gazebo model path environment variables
-```
-cd ~/catkin_ws
-source devel/setup.bash
-bash src/gilbreth/gilbreth_gazebo/scripts/export_gazebo_model_path.bash
-```
+- Source the catkin workspace 
+
+	```
+	cd ~/catkin_ws
+	source devel/setup.bash
+	```
 
 ### View the URDF
-```
-roslaunch urdf_tutorial display.launch model:=`rospack find gilbreth_support`/urdf/gilbreth.xacro
-```
+
+	```
```

Comment:
For some reason your code block didn't register in the markdown. I think you might have to remove the indent to get it to work